test/helpers/BancorFormula.js

```diff
@@ -7,8 +7,8 @@ const { ContractFactory } = require('ethers');
 module.exports.new = async () => {
     const accounts = await ethers.getSigners();
 
-    const abi = fs.readFileSync(path.resolve(__dirname, '../bin/BancorFormula.abi'));
-    const bin = fs.readFileSync(path.resolve(__dirname, '../bin/BancorFormula.bin'));
+    const abi = fs.readFileSync(path.resolve(__dirname, './bin/BancorFormula.abi'));
```

Comment:
What is this file still doing here?

Reply:
probably need to get rid of that

Reply:
We're using this to test the upgrade from a legacy converter. Why should we remove it?